### PR TITLE
[WFCORE-2901] Improve charset usage in CLI tests

### PR DESCRIPTION
**Pull request by marekkopecky**
Improve charset usage in CLI tests

* https://issues.jboss.org/browse/JBEAP-11291
* https://issues.jboss.org/browse/WFCORE-2901

### CODE REVIEW

**Comment by marekkopecky** [comment]
@jfdenise Can you please make a review of this PR?

File: testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CustomCLIExecutor.java [hunk 1]
```diff
@@ -254,8 +255,12 @@ private static String executeProcess(final CliCommandBuilder commandBuilder, fin
         } finally {
             cliProc.destroyForcibly();
         }
-
-        final String cliOutput = out.toString();
+        String cliOutput = null;
+        try {
+            cliOutput = out.toString(StandardCharsets.UTF_8.name());
```
Comment (jfdenise) [inline testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CustomCLIExecutor.java]: On The CLI side, although no interactive session has been started, the CLI will print-out content using the default JVM platform encoding (that can be something different than UTF-8). 
Seems safer to continue to rely on the default encoding.

**Comment by marekkopecky** [comment]
@jfdenise Fixed, keep UTF8 changes only in ModuleTestCase.java

**Comment by marekkopecky** [comment]
retest this please

**Comment by bstansberry** [comment]
retest this please

**Pull request merged**